core/federated/RTI/rti_remote.c

```diff
@@ -1728,7 +1588,7 @@ void initialize_RTI(rti_remote_t* rti) {
   rti_remote->num_feds_proposed_start = 0;
   rti_remote->all_federates_exited = false;
   rti_remote->federation_id = "Unidentified Federation";
-  rti_remote->user_specified_port = 0;
+  rti_remote->user_specified_port = 1;
```

Comment:
Can you explain this change?

Reply:
Sure. Thank you for pointing this out.

The federate and RTI both initialize the `user_specified_port` as 0. 
For the RTI, when the `user_specified_port` is 0, it recognizes that there was no user input to assign a port and assigns the port to the `DEFAULT_PORT`, which is 15045.

For the federate, when the `user_specified_port` is 0, it also recognizes that there was no user input and makes the OS assign the port. 

The `create_TCP_server` gets this input argument `port`, which is actually the `user_specified_port` for both RTI and federate. To distinguish whether the RTI or federate is calling this function, I changed the initial value of the RTI's port as 1.

We need to distinguish whether the RTI or federate called the `create_TCP_server` function for two reasons.
1. As explained before, we should set the default port, which RTI and federate differ.
2. RTI has increments the port number starting from 15045, when the port binding fails. However, the federate does not increment the port number and try binding.

Reply:
Can you add this information in the comment? We want to better understand this decision later. You can even leave a link to this PR in the comment.